### DISCUSSION TITLE
Wikilinks?

### DISCUSSION THREAD

**Discussion by alxlg** [jgm/djot#246] [Ideas]
Since wikilinks, introduced by MediaWiki/Wikipedia, are popular in some Markdown applications like Logseq, Zettlr and Obsidian and even supported by some Markdown parsers, would you be interested in making this:

`[[Foo]]`

an alternative syntax to:

`[Foo](Foo.dj)`

?

Advantages:

1. Make easier to migrate Markdown from those applications to standard Djot.
2. Please users that are used to this syntax.
3. It is more readable.
4. Faster to type.
5. In applications typing `[[` can be used to trigger a dropdown menu with suggestions for existing files.

**Comment by Omikhleia** [comment]
See #26

**Reply by alxlg** [reply]
Thank you, I didn't think about searching in issues.

The TLDR for who don't want to read the whole discussion is: no agreement on link labels like [[foo|label]] and maybe people lost interest since there aren't recent replies.

**Reply by dereuromark** [reply]
POC has been implemented in PHP reference.
